src/components/DataTable/DataTable-story.js

```diff
@@ -0,0 +1,397 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import DataTableRow from '../DataTableRow';
+import DataTableData from '../DataTableData';
+import DataTableRowExpand from '../DataTableRowExpand'; // eslint-disable-line
```

Comment:
Are the `// eslint-disable-line` lines here going to be added in in the future?

Reply:
Nope, those are temporary :-)

Reply:
But wasn't able to push up this PR without fixing lint errors

Reply:
Can always add `--no-verify` if you just want to check stuff in 😄